src/main/java/org/eclipse/jdt/ls/debug/DebugEvent.java

```diff
@@ -0,0 +1,19 @@
+/*******************************************************************************
```

Comment:
Please avoid checking in copies. Let's use symbolic link to reference source code from original repository.